midterm-2023-11-07/code/5040-dara-poposka/styles.css

```diff
@@ -1,41 +1,111 @@
+
```

Comment:
Love the changes, however, the `table` elements have too many problems for actual production usage. The `grid` model is used more often +15 points